pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -321,19 +321,13 @@ func orderByAttributeKeyTags(panelType v3.PanelType, aggregatorOperator v3.Aggre
 	}
 	orderByArray := orderBy(panelType, items, groupTags)
 
-	found := false
-	for i := 0; i < len(orderByArray); i++ {
-		if strings.Compare(orderByArray[i], constants.TIMESTAMP) == 0 {
-			orderByArray[i] = "ts"
-			break
-		}
-	}
-	if !found {
-		if aggregatorOperator == v3.AggregateOperatorNoOp {
+	if panelType == v3.PanelTypeList {
+		if len(orderByArray) == 0 {
```

Comment:
I think you can update this in single line:
`	if panelType == v3.PanelTypeList && len(orderByArray) == 0 {
`

Reply:
No since, if panel type is list but len is not 0 then the next line will execute